lib/vanagon/platform/windows.rb

```diff
@@ -8,7 +8,10 @@ class Windows < Vanagon::Platform
       def generate_package(project)
         target_dir = project.repo ? output_dir(project.repo) : output_dir
         ["mkdir -p output/#{target_dir}",
-        "touch output/#{target_dir}/#{project.name}-#{project.version}-#{project.release}-#{@architecture}.msi"]
+        "mkdir -p #{project.name}",
+        "cp #{project.name}.nuspec #{project.name}/",
+        "(cd #{project.name}; /cygdrive/c/ProgramData/chocolatey/choco pack)",
```

Comment:
:+1: to that.

If there are cases where we do need to use cygwin style paths, I recommend creating a `Shell` type object through which paths can be normalized. So today we're running with cygwin, and we can use `cygpath` tricks if necessary to map between windows (C:/) and cygwin bash (/cygdrive/c) and vice-versa. And in the future we can switch Shells (powershell, msys bash) and only have to change a few places.